poc_mobile_verifier/migrations/3_heartbeats.sql

```diff
@@ -0,0 +1,5 @@
+create table heartbeats (
+       id bytea primary key not null,
+       weight decimal not null,
```

Comment:
nit: Is the `weight` reward_weight? If so can we rename it to `reward_weight` plz?